src/recvmmsg.rs

```diff
@@ -10,7 +10,6 @@ pub const NUM_RCVMMSGS: usize = 16;
 #[cfg(not(target_os = "linux"))]
 pub fn recv_mmsg(socket: &UdpSocket, packets: &mut [Packet]) -> io::Result<usize> {
     let mut i = 0;
-    socket.set_nonblocking(false)?;
```

Comment:
did you wanna write a test?

Reply:
This test is there:
```test_recv_mmsg_multi_iter()```

It should be able to test the correctness of the function. Do you mean a test that checks for timing?

Reply:
👍 
Done

Reply:
Tested it on macos as well.